src/org/intellij/erlang/rebar/importWizard/ImportedOtpApp.java

```diff
@@ -130,25 +133,25 @@ private void addInfoFromRebarConfig() {
     addParseTransformsFromRebarConfig(rebarConfigPsi);
   }
 
-  private void addDependenciesFromAppFile(@NotNull VirtualFile appFile) {
-    ErlangFile appConfigPsi = ErlangTermFileUtil.createPsi(appFile);
-    List<ErlangTupleExpression> applicationDescriptors = ErlangTermFileUtil.getConfigSections(appConfigPsi, "application");
-    ErlangListExpression appAttributes = PsiTreeUtil.getChildOfType(ContainerUtil.getFirstItem(applicationDescriptors), ErlangListExpression.class);
-    ErlangTermFileUtil.processConfigSection(appAttributes, "applications", new Consumer<ErlangExpression>() {
-      @Override
-      public void consume(ErlangExpression deps) {
-        ErlangListExpression dependencyAppsList = deps instanceof ErlangListExpression ? (ErlangListExpression) deps : null;
-        if (dependencyAppsList != null) {
-          for (ErlangExpression depExpression : dependencyAppsList.getExpressionList()) {
-            ErlangQAtom depApp = PsiTreeUtil.getChildOfType(depExpression, ErlangQAtom.class);
-            ErlangAtom appNameAtom = depApp != null ? depApp.getAtom() : null;
-            if (appNameAtom != null) {
-              myDeps.add(appNameAtom.getName());
-            }
+  private void addDependenciesFromAppFile(@NotNull VirtualFile virtualAppFile) {
+    ErlangFile appFile = ErlangTermFileUtil.createPsi(virtualAppFile);
+    ErlangTupleExpression tuple = PsiTreeUtil.getChildOfType(appFile, ErlangTupleExpression.class);
+    ErlangListExpression list = PsiTreeUtil.getChildOfType(tuple, ErlangListExpression.class);
+    List<ErlangTupleExpression> parameters = PsiTreeUtil.getChildrenOfTypeAsList(list, ErlangTupleExpression.class);
+
+    for (ErlangTupleExpression parameter : parameters) {
+      ErlangAtom parameterAtom = PsiTreeUtil.getChildOfType(parameter, ErlangAtom.class);
+      if (parameterAtom != null && "applications".equals(parameterAtom.getName())) {
+        ErlangListExpression applicationList = PsiTreeUtil.getChildOfType(parameter, ErlangListExpression.class);
+        List<ErlangAtom> applicationNames = PsiTreeUtil.getChildrenOfTypeAsList(applicationList, ErlangAtom.class);
+        myDeps.addAll(ContainerUtil.map(applicationNames, new Function<ErlangAtom, String>() {
+          @Override
+          public String fun(ErlangAtom atom) {
+            return atom.getName();
```

Comment:
Because old code depends on old PSI structure and uses many util methods (which are used in many other situations - for config files, for example).
These util methods does not work with new PSI structure. And if I change them, other rebar tests will fail.
So I decide just to change this method.